src/lib/os-release.ts

```diff
@@ -58,3 +58,23 @@ export function getOSVariant(path: string): Promise<string | undefined> {
 export function getOSSemver(path: string): Promise<string | undefined> {
 	return getOSReleaseField(path, 'VERSION');
 }
+
+const L4T_REGEX = /^.*-l4t-r(\d+).(\d+).*$/;
```

Comment:
I do not see any need to capture two groups here. You could just write this regex as:
`/^.*-l4t-r(\d+\.\d+).*$/`
This can simplify the regex operation and simplify the code at line 75.
Also make sure to escape `.` to express the version correctly.